core/src/main/java/org/mule/runtime/core/privileged/processor/AbstractFilteringMessageProcessor.java

```diff
@@ -4,7 +4,7 @@
  * license, a copy of which has been included with this distribution in the
  * LICENSE.txt file.
  */
-package org.mule.runtime.core.processor;
+package org.mule.runtime.core.privileged.processor;
```

Comment:
by moving ProcessIfStartedMessageProcessor, you can now move this to compatibility too